scripts/summarize_go_test_json.py

```
@@ -37,15 +47,29 @@ def parse_events(lines: list[str]) -> dict[str, PackageResult]:
         result = results.setdefault(package, PackageResult())
         action = event.get("Action")
+        test = event.get("Test")
+        test_result = None
+        if isinstance(test, str) and test != "":
+            test_result = result.tests.setdefault(test, TestResult())
+
         if action == "output":
             output = event.get("Output")
             if isinstance(output, str):
-                result.output.append(output.rstrip("\n"))
+                line = output.rstrip("\n")
```

Ruff (0.15.20)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)